README.md

```diff
@@ -112,10 +108,10 @@ metadata:
 spec:
   deploymentMode: dev
   runtime: spring-boot
-  version: 2.1.16
+  version: 2.1.16   // runtime version
```

Comment:
Do comments work like this in YAML?

Reply:
No, but it's just meant as a call-out in the example… Best to remove it?

Reply:
Whatever you think it best. I am only thinking that people might copy-paste it and it won't work